WooCommerce/UITestsFoundation/Screens/Orders/OrdersScreen.swift

```diff
@@ -17,17 +26,20 @@ public final class OrdersScreen: ScreenObject {
         (try? OrdersScreen().isLoaded) ?? false
     }
 
-    init(app: XCUIApplication = XCUIApplication()) throws {
+    public init(app: XCUIApplication = XCUIApplication()) throws {
```

Comment:
Once the local `isLoaded` is removed, and the `testGotoOrdersScreen` will now refer to `isLoaded` from `ScreenObject`, there's no need to keep this public:

```suggestion
   init(app: XCUIApplication = XCUIApplication()) throws {
```

Reply:
I was getting this error `'OrdersScreen' initializer is inaccessible due to 'internal' protection level` when building the app earlier, adding `public` seems to have fixed it. Will keep this here for now since it is harmless.

Reply:
Yes, this is because you were using the local `isLoaded` previously. But as you say, it's not harmful to keep it this was 👍